sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Models/SlackChannelProperties.cs

```diff
@@ -47,7 +46,8 @@ public SlackChannelProperties()
         /// True.</param>
         /// <param name="isValidated">Whether this channel is validated for the
         /// bot</param>
-        public SlackChannelProperties(string clientId, string clientSecret, string verificationToken, bool isEnabled, string landingPageUrl = default(string), string redirectAction = default(string), string lastSubmissionId = default(string), bool? registerBeforeOAuthFlow = default(bool?), bool? isValidated = default(bool?))
+        /// <param name="signingSecret">The Slack signing secret.</param>
+        public SlackChannelProperties(bool isEnabled, string clientId = default(string), string clientSecret = default(string), string verificationToken = default(string), string landingPageUrl = default(string), string redirectAction = default(string), string lastSubmissionId = default(string), bool? registerBeforeOAuthFlow = default(bool?), bool? isValidated = default(bool?), string signingSecret = default(string))
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?